### PR TITLE
Map style fixes

### PR DESCRIPTION
**Pull request by danieldunderfelt**
PR for MAP-582 and MAP-586.
The cycleway color is a bit random, any ideas what it should be?

### CODE REVIEW

File: hsl-gl-map-v9-base.json [hunk 1]
```diff
@@ -1916,7 +1916,7 @@
                 "line-join": "round"
             },
             "paint": {
-                "line-color": "#fff",
+                "line-color": "#ffffce",
```
Comment (varpushaukka) [inline hsl-gl-map-v9-base.json]: This doesn't look good. Maybe we should make new (stricter) style layer for cycle ways. When you zoom in Kamppi for example, the areas tagged with bicyle=yes are surrounded with yellow lines.

File: hsl-gl-map-v9-base.json [hunk 2]
```diff
@@ -3174,6 +3186,126 @@
...
+                            0.5
+                        ],
+                        [
+                            20,
+                            40
+                        ]
+                    ]
+                }
+            }
+        },
```
Comment (varpushaukka) [inline hsl-gl-map-v9-base.json]: Nice! This fixex the problem! :)

File: hsl-gl-map-v9-base.json (hunk 1 above)
Comment (varpushaukka) [reply hsl-gl-map-v9-base.json]: I think #42 is the correct answer

Comment (danieldunderfelt) [reply hsl-gl-map-v9-base.json]: Well, I had no specs about how they should be different, just that they should be different. I'm good with #42, so I'll revert this part of the PR.

**Pull request merged**